spec/services/reporting/account_reuse_report_spec.rb

```diff
@@ -245,11 +248,19 @@
             verified_at: user[:sp_timestamp][i],
           )
         end
+
+        if user[:id] == 21
+          user_to_delete = User.create({ id: user[:id] })
+          DeletedUser.create_from_user(user_to_delete)
+          user_to_delete.destroy!
+        else
+          User.create({ id: user[:id] })
+        end
```

Comment:
I think it's a bad pattern to create a user with a specific ID in specs because the ID column is auto-incrementing, and the test could create the wrong one.

Another option could be to create an array of users and delete one specific one:

```
users = 30.times.map { create(:user) }
to_delete = users[21]
users[21] = DeletedUser.create_from_user(to_delete)
to_delete.destroy
```

Reply:
Ah - okay,  I was attempting to connect these users to the identities that I created earlier, but I can make it more generic...